typescript/phoenix-sdk/src/token.ts

```diff
@@ -0,0 +1,65 @@
+import { AccountLayout, TOKEN_PROGRAM_ID } from "@solana/spl-token";
+import { Connection, PublicKey, TokenAmount } from "@solana/web3.js";
+
+import { TokenParams } from "./types";
+
+export class Token {
+	name: string;
+	symbol: string;
+	logoUri: string;
+	balance: TokenAmount | null;
```

Comment:
feels kind of odd for this class to own the balance for a particular user. do you get weird behavior if the user switches wallet?

Reply:
if the app switches wallet then it will re-instantiate the Client() class so everything is wiped and refetched